MDHelpers/MDStatics.cs

```diff
@@ -279,6 +286,18 @@ public static List<MemberInfo> GetTypeMemberInfos(Type ObjectType)
             return DeDupedMembers;
         }
 
+        /// <summary>Creates an instance of the type based on the base class T</summary>
```

Comment:
The XML documentation plugin we use also generate parameter documentation, should probably add this as well